config/datastores/posixDatastore.yaml

```diff
@@ -35,3 +35,5 @@ datastore:
     Background: lsst.daf.butler.formatters.fitsCatalogFormatter.FitsCatalogFormatter
     Config: lsst.daf.butler.formatters.pexConfigFormatter.PexConfigFormatter
     Packages: lsst.daf.butler.formatters.pickleFormatter.PickleFormatter
+    PropertyList: lsst.daf.butler.formatters.pickleFormatter.PickleFormatter
+    PropertySet: lsst.daf.butler.formatters.pickleFormatter.PickleFormatter
```

Comment:
Probably not an issue for this ticket, but we should have and use a YAML formatter for these.

Reply:
Oh, I hadn't realized the YAML I/O was not in daf_base.  I'd definitely like to see it moved there, though I wouldn't object to having daf_butler depend on daf_persistence temporarily.